python/cuml/cuml/ensemble/randomforest_common.pyx

```diff
@@ -421,7 +421,7 @@ class BaseRandomForestModel(Base, InteropMixin):
         cdef uintptr_t y_ptr = y.ptr
         cdef int n_rows = X.shape[0]
         cdef int n_cols = X.shape[1]
-        cdef level_enum verbose = <level_enum> self.verbose
+        cdef level_enum verbose = <level_enum> self._verbose_level
```

Comment:
As described in #6394, anywhere we're forwarding a log level to a `libcuml` call is doing the wrong thing. I have a branch locally that eventually I'll get around to finishing up, but in short - passing this per-call should go away in favor of contextmanager API for wrapping libcuml calls (ensuring the log level doesn't persist after a single call).